tests/test_plotting.py

```diff
@@ -717,12 +718,14 @@ def test_plot_boozer_surface(DSHAPE_current):
 @pytest.mark.unit
 @pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_plot_qs_error(DSHAPE_current):
+def test_plot_qs_error():
     """Test plotting qs error metrics."""
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
-    fig, ax, data = plot_qs_error(eq, helicity=(0, 0), log=False, return_data=True)
+    eq = get("WISTELL-A")
+    fig, ax, data = plot_qs_error(eq, helicity=(1, eq.NFP), log=False, return_data=True)
     for string in ["rho", "f_T", "f_B", "f_C"]:
         assert string in data.keys()
+        if string != "rho":
```

Comment:
might want to include a comment about why we're doing this test, out of context it looks kind of weird